### ISSUE TITLE
Suggestions for deployment guide clarity

### ISSUE THREAD

**Issue by kriszentner** [OfficeDev/microsoft-teams-apps-greatideas#11]
Hi Folks,

I've gone through the [deployment guide](https://github.com/OfficeDev/microsoft-teams-apps-greatideas/wiki/Deployment-guide), and have a couple suggestions after going through it from beginning to end:

## Step 3, number 10 and 15:
This references `api://fully-qualified-domain-name.com/{BotID}`, and `api://subdomain.example.com:6789/c6c1f32b-5e55-4997-881a-753cc1d563b7/access_as_user` 

It seems like both of these should just be the Azure Front Door URI (at least this is what I used). Otherwise it's difficult to know where `fully-qualified-domain-name.com` is coming from. Also the use of the nonstandard port seem to imply something to the user that is not explicit. It would be good to better explain this, or just use the same url of `[BaseResourceName].azurefd.net` as in the rest of the documentation.

## Step 4
This is *almost* obvious, but it would be good to tell the user to clone the git repo, where they'll find the `manifest.json` file. This wasn't immediately clear to me.

**Comment by maxi07** [comment]
I agree, it's quite hard to follow the instructions. Also, the `api://<<applicationurl>>/<<botId>>` in the _webApplicationInfo_ section is not present in the current manifest.json build. How shall one handle this?
